wemake_python_styleguide/violations/complexity.py

```diff
@@ -1095,3 +1097,38 @@ class TooComplexAnnotationViolation(ASTViolation):
 
     error_template = 'Found too complex annotation: {0}'
     code = 234
+
+
+@final
+class TooManyImportedModuleMembersViolation(ASTViolation):
+    """
+    Forbids ``from ... import ...`` with too many imported names.
+
+    Reasoning:
+        Importing too many names from one import is easy way to cause
+        `WPS203`.
```

Comment:
We use double backtics.

Reply:
And I also suggest to add more detailed description: even I don't remember what `WPS203` is 🙂